packages/flutter/test/material/slider_test.dart

```diff
@@ -1400,7 +1395,13 @@ void main() {
                   flags: <SemanticsFlag>[SemanticsFlag.scopesRoute],
                   children: <TestSemantics>[
                     TestSemantics(
-                      id: 5,
+                      id: 3,
+                      flags: <SemanticsFlag>[SemanticsFlag.hasEnabledState],
+                      actions: <SemanticsAction>[SemanticsAction.increase, SemanticsAction.decrease],
```

Comment:
(see earlier comment): Since the slider is disabled I would have expected it not to have these actions. AFAIK a disabled button also doesn't publish a tap action.

Reply:
Removed.